ports/nrf/boards/pca10056/mpconfigboard_s140.mk

```diff
@@ -0,0 +1,8 @@
+MCU_SERIES = m4
+MCU_VARIANT = nrf52
+MCU_SUB_VARIANT = nrf52840
+SOFTDEV_VERSION = 6.0.0-6.alpha
+
+LD_FILE = boards/nrf52840_1M_256k_s140_$(SOFTDEV_VERSION).ld
```

Comment:
A custom linker script would perhaps be better here, following the `feather52` example, since we will need to use a modified version of the serial bootloader in addition to the J-Link. I guess we don't technically need the serial bootloader on the PCA10056 since it has an on board J-Link, but we can still test the serial bootloader as an option on this board until commercial modules are available with the final silicon.

Reply:
Not sure how to proceed, should I leave as is until a new bootloader is available?

Reply:
Don't worry about it ... I'll put the new linker script in myself when I generate the custom bootloader, since I'm not sure of the exact sizes right now anyway. I've only built it against s132.